src/Zynga/Framework/PgData/V1/PgCachedResultSet.hh

```diff
@@ -37,8 +41,9 @@ class PgCachedResultSet<Tv as TypeInterface> extends Base<Tv> {
     $rawType = $this->_rawType;
     $typeChecksum = md5($this->_rawType);
     $whereChecksum = $this->_where->createWhereChecksum();
+    $modelChecksum = md5($this->_model);
 
-    $this->_checksum = $typeChecksum.'|'.$whereChecksum;
+    $this->_checksum = $typeChecksum.'|'.$whereChecksum.'|'.$modelChecksum;
```

Comment:
I believe each md5 checksum holds 32 bits ex: 098f6bcd4621d373cade4e832627b4f6
So our key becoming 32+32+32.
It may reduce size to 32 bits if we do md5 at one shot as md5($this->_rawType + where + model)